src/Php72/Php72.php

```diff
@@ -60,4 +61,23 @@ public static function utf8_decode($s)
 
         return substr($s, 0, $j);
     }
+
+    public static function php_os_family()
+    {
+        if ('\\' === DIRECTORY_SEPARATOR) {
+            return 'Windows';
+        }
+
+        $map = array(
+            'Darwin' => 'Mac',
```

Comment:
and before to Mac... http://git.php.net/?p=php-src.git;a=commitdiff;h=362d2e42a02fe018a7d1261a53ff59b73fed91f6

they cannot decide - osx, mac, darwin now... changing